modules/core/src/test/java/org/apache/ignite/internal/util/StripedExecutorTest.java

```diff
@@ -140,6 +141,22 @@ public void testQueueSize() throws Exception {
         assertEquals(1, stripedExecSvc.queueSize());
     }
 
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testStarvationDetected() throws Exception {
```

Comment:
Unfortunately this test is flaky. It will sometimes fail for the 1st time and almost always will fail for 2nd run.

Reply:
I've fixed it, ~200 sequential runs -> pass